### DISCUSSION TITLE
Audio not being picked up in OBS

### DISCUSSION THREAD

**Discussion by n00idea** [ExistentialAudio/BlackHole#583] [Q&A]
Hi, 

I installed blackhole - all was working fine yesterday, I opened my OBS today and now the internal (desktop) audio isn't being picked up.

Device for sounds output is set to my new audio devices channel I made and I can hear audio being played through my headphones when it's selected but nothing being picked up by OBS

I have uninstalled and reinstalled still no luck

Any help would greatly be appreciated,

Ty

**Comment by n00idea** [comment]
SOLVED FOR NOW

For anyone having the same issue - this isn't really a solution but I toggled on/off sounds preferences channel and the audio devices channels to use and restarted my mac a bunch of time and it just started to work..

**Comment by Lacrowx** [comment]
I have the same issue and can't get OBS to pick up any sound for the life of me. I have followed all the instructions. Have you found another workaround? 

![image](https://user-images.githubusercontent.com/45010439/169661389-1bb0f8d5-7255-4f9f-8453-d5932ebca5fa.png)

**Comment by n00idea** [comment]
I can only suggest what I tried above - haven't had any problems since..
